chat-plugins/dexsearch.js

```diff
@@ -31,8 +31,11 @@ if (!process.send) {
 		ds: 'dexsearch',
 		dsearch: 'dexsearch',
 		dexsearch: function (target, room, user, connection, cmd, message) {
-			if (!this.canBroadcast()) return;
-
+			if (this.broadcasting && !user.can('broadcast', null, room)) {
```

Comment:
We might be better off adding another argument to `canBroadcast` though. Or adding another possible case to the existing argument.